pyproject.toml

```diff
@@ -7,7 +7,7 @@ requires = [
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools]
-packages = ["oar"]
+packages = ["oar", "tools"]
```

Comment:
do we need new pkg `tools` here?

Reply:
yes, the jira_notificator file is in the tools directory. If it is not a package, an error is displayed:
```
Traceback (most recent call last):
  File "/Users/tdavid/Projects/release-tests/venv/bin/oarctl", line 5, in <module>
    from oar.cli.cmd_controller_group import cli
  File "/Users/tdavid/Projects/release-tests/oar/cli/cmd_controller_group.py", line 10, in <module>
    from tools.jira_notificator import jira_notificator
ModuleNotFoundError: No module named 'tools'
```

or I can move the script to the oar package.

Reply:
yes, I think you can move it to `oar/controller`